Makefile

```diff
@@ -213,17 +226,43 @@ update-codegen:
 	$(call run-informer-gen,)
 .PHONY: update-codegen
 
+update-crd:
+	$(CONTROLLER_GEN) crd paths="$(GO_PACKAGES)" output:crd:stdout > ./deploy/operator/00_scylla.scylladb.com_scyllaclusters.yaml
+.PHONY: update-crd
+
+update-dev-deploy:
+	mkdir -p deploy/manager/dev
+	cp deploy/manager/*.yaml deploy/manager/dev/
```

Comment:
dev it's a special case, i would like to leave prod manifests in higher directory. It would be consistent with operator which currently doesn't have dev and prod distinction

Reply:
I don't think that's hierarchically correct. dev isn't a subcase of prod, both are on the same level. To the analogy with operator, that's a different case because there is one type only (prod), so it doesn't need nested dirs like when there are 2 (or more) types of deployment.

Reply:
say you want to recursively copy all prod manifests, I don't think it would be expected to copy dev ones as well